1.1/spec/11-Part-08.adoc

```diff
@@ -24,13 +24,13 @@ SFA-CA also describes M coordinate values that may be part of geometry encodings
 
-=== Influence of Coordinate Reference Systems on geometric computations
-Geometric computations must always be mindful of the kind of space that is described by a Coordinate Reference System (CRS). A Geometry object consists of a set of coordinates and a specification on how the coordinates should be interpreted: the CRS. Taken together, coordinates and CRS allow performing computations on Geometry objects. For example, sizes can be calculated or new Geometry objects can be created. Some Coordinate Reference Systems describe a two-dimensional flat space. In that case, coordinates are understood to be Cartesian, and Cartesian geometric computations can be performed. But Coordinate Reference Systems can describe other types of spaces, to which Cartesian computations are not applicable. For example, if CRS http://www.opengis.net/def/crs/OGC/1.3/CRS84[`+<http://www.opengis.net/def/crs/OGC/1.3/CRS84>+`] is used, coordinates are to be interpreted as degrees of latitude and longitude, designating positions on a spheroid. The distance between two points using this CRS is different from the distance between two points that have the same coordinates but are based on a Cartesian CRS.
+=== Influence of Reference Systems on computations
+Geometric computations must always be mindful of the kind of space that is described by a Spatial Reference System (SRS). A Geometry object consists of a set of coordinates and a specification on how the coordinates should be interpreted: the SRS, commonly a CRS. Taken together, coordinates and SRS allow performing computations on Geometry objects. For example, sizes can be calculated or new Geometry objects can be created. Some Spatial Reference Systems describe a two-dimensional flat space. In that case, coordinates are understood to be Cartesian, and Cartesian geometric computations can be performed. But Spatial Reference Systems can describe other types of spaces, to which Cartesian computations are not applicable. For example, if CRS http://www.opengis.net/def/crs/OGC/1.3/CRS84[`+<http://www.opengis.net/def/crs/OGC/1.3/CRS84>+`] is used, coordinates are to be interpreted as degrees of latitude and longitude, designating positions on a spheroid. The distance between two points using this CRS is different from the distance between two points that have the same coordinates but are based on a Cartesian CRS or other SRS.
 
-To avoid erroneous computations involving Geometry, data publishers are recommended to clearly indicate the type of space that is described by the CRS.
+To avoid erroneous computations involving Geometry, data publishers are recommended to clearly indicate the type of space that is described by the SRS/CRS.
```

Comment:
Is the term "CRS" superfluous here given the context set that SRS is the broader team?